### DISCUSSION TITLE
Nested Taxonomies

### DISCUSSION THREAD

**Discussion by pathly-dev** [statamic/cms#5228] [Q&A] [answered]
Hey there,

I use taxonomies as tags for selection. So each blog post of a blog has a specific taxonomy term, so the user can select for specific terms in the frontend and get all the matching posts that are tagged with this term.
Now I also need sub terms. I have a structre like the following:

- term a
   - term aa
   - term ab
- term b
- term c

I already found this post https://statamic.com/forum/4106-nested-taxonomies and tried to put a field of type taxonomy terms in the taxonmy blueprint and a toggle if the term is parent, if yes than added the sub terms to that term. But it's not working like I thought. In the panel I still can only choose the parent, but nothing more. But I want, that if you choose a parent that you can also choose one of the children.

**Comment by jasonvarga** [comment, marked as answer]
It's one of the most upvoted feature requests. Go ahead and give it another thumbs up.

https://github.com/statamic/ideas/issues/92
